core/storage/feedback/gae_models.py

```diff
@@ -113,14 +113,30 @@ def get_by_exp_and_thread_id(cls, exploration_id, thread_id):
 
     @classmethod
     def get_threads(cls, exploration_id):
-        """Returns an array of threads associated to the exploration.
+        """Returns threads ordered by last updated associated to the
```

Comment:
Returns a list of threads associated to the exploration, ordered by their last-updated field.